code/messaging/src/main/java/com/adobe/marketing/mobile/aepuitemplates/utils/AepUIConstants.kt

```diff
@@ -0,0 +1,99 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.aepuitemplates.utils
+
+/**
+ * Object containing constant values used throughout the application.
+ *
+ * This class is structured to hold constants related to card templates, including schema data,
+ * UI elements, and specific styling details. These constants are referenced in various UI model
+ * classes to ensure consistency and avoid hardcoding string values.
+ */
+class AepUIConstants {
+
+    class CardTemplate {
+
+        class SchemaData {
+            companion object {
+                const val TITLE = "title" // Key for the title in the schema
+                const val BODY = "body" // Key for the body text in the schema
+                const val IMAGE = "image" // Key for the image in the schema
+                const val ACTION_URL = "actionUrl" // Key for the action URL in the schema
+                const val BUTTONS = "buttons" // Key for the buttons in the schema
+                const val DISMISS_BTN = "dismissBtn" // Key for the dismiss button in the schema
+            }
+
+            class Meta {
+                companion object {
+                    const val ADOBE_DATA = "adobe" // Key for Adobe-specific data
+                    const val TEMPLATE = "template" // Key for template information
+                }
+            }
+        }
+
+        class DismissButton {
```

Comment:
Can we move this inside UI element? Guessing we need an `AEPDismissButton` model for that too?